generator/sbpg/generator.py

```diff
@@ -34,9 +34,6 @@ def get_args():
   parser.add_argument('--python',
                       action="store_true",
                       help='Target language: Python.')
-  parser.add_argument('--python_jit',
```

Comment:
Is it worthwhile keeping this and throwing a meaningful error to the user?

Reply:
No need to keep this around, the generator is only used by internal people and are not usually invoked by hand (just via the Makefile)